server/passport-providers.js

```diff
@@ -159,5 +159,20 @@ module.exports = {
     scope: ['email'],
     link: true,
     failureFlash: true
+  },
+  'github-repo': {
```

Comment:
This is there, as we need repo scope, but we don't want to ask users for it if they arn't planning on using the feature